pkg/gluetikv/glue.go

```diff
@@ -55,3 +55,8 @@ func (Glue) StartProgress(ctx context.Context, cmdName string, total int64, redi
 func (Glue) Record(name string, val uint64) {
 	summary.CollectUint(name, val)
 }
+
+// GetVersion implements glue.Glue.
+func (Glue) GetVersion() string {
+	return "BR\n" + utils.BRInfo()
+}
```

Comment:
Please add a unit test to make sure the version matches a certain pattern.

Reply:
addressed in https://github.com/pingcap/br/pull/734/commits/7b4365c010352b1c32d5639cb4ce366876692fbe